language/move-prover/bytecode/src/function_target_pipeline.rs

```diff
@@ -20,7 +21,7 @@ pub struct FunctionTargetsHolder {
 }
 
 /// Describes a function target variant.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
I guess we could keep the `Copy` here but I am fine either way

Reply:
This prepares for FunctionVariant more than just a ground enum.